baby-gru/src/components/BabyGruMolecule.js

```diff
@@ -28,6 +28,16 @@ export function BabyGruMolecule(commandCentre) {
 };
 
 
+BabyGruMolecule.prototype.delete = async function (gl) {
+    const $this = this
+    Object.getOwnPropertyNames(this.displayObjects).forEach(displayObject => {
+        if(this.displayObjects[displayObject].length > 0) {this.hide(displayObject, gl)}
+    })
```

Comment:
@stuartjamesmcnicholas this is where the method `BabyGruMolecule.hide` gets called. The definition of that method is a a few lines later, and it looks like this:
```javascript
BabyGruMolecule.prototype.hide = function (style, gl) {
    this.displayObjects[style].forEach(displayBuffer => {
        displayBuffer.visible = false
    })
    gl.current.drawScene()
}
```